subjects/buzzinga/README.md

```diff
@@ -0,0 +1,53 @@
+# buzzinga
+
+### Instructions
+Write a function named `BuzZinga()` that takes a number as an argument and prints the following:
+    
```

Comment:
You need to say what the number 0 does...

Reply:
and how negative works

Reply:
done